src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/ResultHelper.cs

```diff
@@ -0,0 +1,181 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Classes;
+using Microsoft.PowerToys.Run.Plugin.WindowsSettings.Properties;
+using Wox.Plugin;
+using Wox.Plugin.Logger;
+
+namespace Microsoft.PowerToys.Run.Plugin.WindowsRegistry.Helper
+{
+    /// <summary>
+    /// Helper class to easier work with results
+    /// </summary>
+    internal static class ResultHelper
+    {
+        /// <summary>
+        /// Return a list with <see cref="Result"/>s, based on the given list.
+        /// </summary>
+        /// <param name="list">The original result list to convert.</param>
+        /// <param name="iconPath">The path to the icon of each entry.</param>
+        /// <returns>A list with <see cref="Result"/>.</returns>
+        internal static List<Result> GetResultList(
+            in IEnumerable<IWindowsSetting> list,
+            string query,
+            in string iconPath)
+        {
+            var resultList = new List<Result>(list.Count());
+
+            foreach (var entry in list)
+            {
+                var type = entry.Command.StartsWith("ms-settings", StringComparison.InvariantCultureIgnoreCase)
+                    ? Resources.SettingsApp
+                    : Resources.ControlPanel;
+
+                var result = new Result
+                {
+                    Action = (_) => DoOpenSettingsAction(entry),
+                    IcoPath = iconPath,
+                    SubTitle = $"{Resources.Area}: {entry.Area} - {Resources.Type}: {type}",
```

Comment:
**Suggest to change subtitle like this:**

```suggestion
                    SubTitle = $"{Resources.Area} \"{entry.Area}\" {Resources.In} {type}",
```


![image](https://user-images.githubusercontent.com/61519853/116818736-2fffa480-ab6d-11eb-8345-21feff1b5c94.png)

Reply:
Added